rosdep/python.yaml

```
@@ -5899,6 +5899,13 @@ python3-empy:
   openembedded: [python3-empy@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-empy']
   ubuntu: [python3-empy]
+python3-evdev:
+  debian: [python3-evdev]
+  fedora: [python3-evdev]
```

Here's the equivalent Gentoo package: https://packages.gentoo.org/packages/dev-python/python-evdev
```suggestion
  fedora: [python3-evdev]
  gentoo: [dev-python/python-evdev]
```

---

This is the same as the one for the Python 2 package then? Should we remove this from this one?

https://github.com/ros/rosdistro/blob/45fdf02ae88ea2116fba6f390f5d61fd70a626b7/rosdep/python.yaml#L1507

---

No like Arch Gentoo moved to python3 by default already. All python rules on Arch and Gentoo actually resolve to python3.